mount/main.go

```diff
@@ -0,0 +1,70 @@
+//  Copyright 2021 Authors of Cilium
```

Comment:
maybe this could live under `tools/` directory?

Reply:
Yes but this will start cluttering the root repository. Although those programs are invoked using `go run` it should not be a requirement for them to live under `/tools`. `bugtool` should likely be under that directory as well.

Reply:
Yeah, that's a fair point. I'll move it under `tools`.